app.py

```diff
@@ -57,6 +57,19 @@ def save_images(visuals, image_path, aspect_ratio=1.0, width=256):
 def index():
     return "Color Wave Backend"
 
+#Takes image and returns.
+@app.route('/upload', methods=['POST'])
+def upload():
+    sketchBase64 = request.form["sketch"]
+    sketch = Image.open(BytesIO(base64.b64decode(sketchBase64)))
+
+
+    img_io = BytesIO()
+    sketch.save(img_io, 'JPEG', quality=70)
+    img_io.seek(0)
+    
+return send_file(img_io, mimetype='image/gif')
+
```

Comment:
As mentioned yesterday, the majority of this logic is done. See the endpoint below this called `/image`. It does the exact same thing except runs it through the model